website/source/docs/providers/aws/r/cloudformation_stack.html.markdown

```diff
@@ -15,13 +15,23 @@ Provides a CloudFormation Stack resource.
 ```
 resource "aws_cloudformation_stack" "network" {
   name = "networking-stack"
+  parameters = {
```

Comment:
This can just be updated to:

```
  parameters {
    VpcCIDR = "10.0.0.0/16"
  }
```

Notice the lack of need for the `=` in the parameter declaration block and then the name of the parameter